src/arch/riscv/isa/templates/vector_mem.isa

```diff
@@ -452,8 +466,7 @@ def template VlmConstructor {{
     %(set_reg_idx_arr)s;
     %(constructor)s;
 
-    const uint32_t micro_vlmax = vlen / width_EEW(_machInst.width);
-    int32_t micro_vl = (std::min(this->vl, micro_vlmax) + 7) / 8;
+    int32_t micro_vl = (this->vl + 7) / 8;
```

Comment:
I think we can separate the `fix vlm.v/vsm.v` into different PR. How do you think

Reply:
Done #830.